projects/plugins/jetpack/_inc/client/performance/speed-up-site.jsx

```diff
@@ -234,55 +236,68 @@ const SpeedUpSite = withModuleSettingsFormHelpers(
 					module="performance-speed"
 				>
 					{ ( foundPhoton || foundAssetCdn ) && (
-						<SettingsGroup
-							hasChild
-							support={ {
-								link: getRedirectUrl( 'jetpack-support-site-accelerator' ),
-							} }
-						>
-							<p>
-								{ __(
-									'Load pages faster by allowing Jetpack to optimize your images and serve your images and static files (like CSS and JavaScript) from our global network of servers.',
-									'jetpack'
+						<>
+							<SettingsGroup
+								hasChild
+								support={ {
+									link: getRedirectUrl( 'jetpack-support-site-accelerator' ),
+								} }
+							>
+								<p>
+									{ __(
+										'Load pages faster by allowing Jetpack to optimize your images and serve your images and static files (like CSS and JavaScript) from our global network of servers.',
+										'jetpack'
+									) }
+								</p>
+								{ canAppearInSearch && (
+									<ToggleControl
+										checked={ siteAcceleratorStatus }
+										toggling={ togglingSiteAccelerator }
+										onChange={ this.handleSiteAcceleratorChange }
+										disabled={ ! canDisplaySiteAcceleratorSettings }
+										label={ __( 'Enable site accelerator', 'jetpack' ) }
+									/>
 								) }
-							</p>
-							{ canAppearInSearch && (
-								<ToggleControl
-									checked={ siteAcceleratorStatus }
-									toggling={ togglingSiteAccelerator }
-									onChange={ this.handleSiteAcceleratorChange }
-									disabled={ ! canDisplaySiteAcceleratorSettings }
-									label={ __( 'Enable site accelerator', 'jetpack' ) }
-								/>
+								<FormFieldset>
+									{ foundPhoton && (
+										<ModuleToggle
+											slug="photon"
+											disabled={ this.props.isUnavailableInOfflineMode( 'photon' ) }
+											activated={ this.props.getOptionValue( 'photon' ) }
+											toggling={ this.props.isSavingAnyOption( 'photon' ) }
+											toggleModule={ this.toggleModule }
+										>
+											<span className="jp-form-toggle-explanation">
+												{ __( 'Speed up image load times', 'jetpack' ) }
+											</span>
+										</ModuleToggle>
+									) }
+									{ foundAssetCdn && (
+										<ModuleToggle
+											slug="photon-cdn"
+											activated={ this.props.getOptionValue( 'photon-cdn' ) }
+											toggling={ this.props.isSavingAnyOption( 'photon-cdn' ) }
+											toggleModule={ this.toggleModule }
+										>
+											<span className="jp-form-toggle-explanation">
+												{ __( 'Speed up static file load times', 'jetpack' ) }
+											</span>
+										</ModuleToggle>
+									) }
+								</FormFieldset>
+							</SettingsGroup>
+							{ this.props.isAtomicSite && this.props.isPageOptimizeActive && (
```

Comment:
Do you think whether checking the value of `isPageOptimizeActive` is enough?

Reply:
That's a good question! We only display this link here because we removed Settings > Performance for Atomic sites. For non-WoA sites, Settings > Performance will still exist (created by the Page Optimize plugin), so we don't need to display it again here. Does it make sense?

Reply:
The link seems to be additional information in the “Performance & speed” section but we still display that section on both “Settings > Performance” and “Jetpack > Settings > Performance” for non-WoA sites.

I'm fine with both 🙂